packages/babel-plugin-transform-react-constant-elements/test/fixtures/constant-elements/destructure/expected.js

```diff
@@ -4,8 +4,10 @@ class AnchorLink extends Component {
     const isExternal = _props.isExternal,
           children = _props.children;
 
+    var _ref = <a>{children}</a>;
+
     if (isExternal) {
-      return <a>{children}</a>;
+      return _ref;
```

Comment:
This looks like a regression (`React.createElement` now unconditionally called, with result potentially discarded)